src/content/docs/extensions/llm.mdx

```diff
@@ -65,9 +65,7 @@ CALL CREATE_EMBEDDING(
     <PROMPT>,
     <PROVIDER>,
     <MODEL>,
-    dimensions := <INT64>,
-    region := <STRING>,
-    endpoint := 'http://localhost:11434'
+    -- optional params
```

Comment:
How about we list all the variants here, same as https://dev-docs.kuzudb.com/extensions/json/#json-functions

Reply:
Ah yes, I notice that in the examples below, we only provide examples for the required params. Not the optional params.

Reply:
Maybe replace one of the examples in the list with an example that contains an optional parameter using `:=`? @tgahunia05